src/Rendering/edgesRenderer.ts

```diff
@@ -447,7 +469,7 @@ export class EdgesRenderer implements IEdgesRenderer {
      * @return true if ready, otherwise false.
      */
     public isReady(): boolean {
-        return this._lineShader.isReady();
+        return this._lineShader.isReady(this._source, this._source.hasThinInstances);

```

Comment:
Are we sure it is hasTHIN and not regular instances instead?

Reply:
The PR does not support regular  instances, see my comments in #6539

Reply:
oh ok I did not read that. And how will it looks like from the user standpoint? what should I do to get instancing for my edge renderers?

Reply:
~~You mean, "what should I do to get edge renderer for my instances"?~~

~~Nothing changed, except that a shader material is not created for each edge renderer, only a single one is created by scene.~~

Ok, I may have found a way, as I can see the current implementation for edge rendering for instances does not account for visible sub meshes, anyway.

Going to commit some changes.

Reply:
Committed.

I have added a new `edgesShareWithInstances` property on the mesh, to let the user tell if they want to use the edge renderer of the source mesh for all instances or not.